src/components/clusters/ClusterDetailsMultiRegion/components/AccessRequest/components/AccessRequestDetails.tsx

```diff
@@ -56,22 +60,35 @@ const AccessRequestDetails = ({ accessRequest }: AccessRequestDetailsProps) => {
           <DescriptionListGroup>
             <DescriptionListTerm>Created Time</DescriptionListTerm>
             <DescriptionListDescription>
-              <Tooltip
-                content={
-                  <span>
-                    Last updated on{' '}
-                    {accessRequest.updated_at
-                      ? new Date(accessRequest.updated_at).toLocaleDateString()
-                      : 'N/A'}
-                  </span>
-                }
+              <Timestamp
+                date={new Date(accessRequest?.created_at || '')}
+                dateFormat={TimestampFormat.short}
+                // timeFormat={TimestampFormat.long}
+                is12Hour={false}
+                locale="en-CA"
+                // shouldDisplayUTC
+                tooltip={{
+                  variant: TimestampTooltipVariant.custom,
+                  tooltipProps: {
+                    className: 'openshift',
+                  },
+                  content: (
+                    <>
+                      Last updated on{' '}
+                      <Timestamp
+                        date={new Date(accessRequest?.updated_at || '')}
+                        dateFormat={TimestampFormat.short}
+                        // timeFormat={TimestampFormat.long}
+                        is12Hour={false}
+                        locale="en-CA"
+                        // shouldDisplayUTC
```

Comment:
Why are we hardcoding locale="en-CA"?
"ShouldDisplayUTC" comment?

Reply:
This is what is designed as part of OCMUI-3585 which will be worked on soon.  The designs display a time so I coded it the way OCMUI-3585 wants and commented out code for items we don't display now. AKA the code will be uncommented out as soon as OCMUI-3585 is started.